clippy_lints/src/methods/option_filter_map.rs

```diff
@@ -0,0 +1,21 @@
+use crate::utils::{match_qpath, paths};
+use rustc_hir::{self, Expr, ExprKind};
+use std::iter;
+
+fn calls_option_method<'tcx>(expr: &'tcx [Expr<'_>], fn_name: &str) -> bool {
+    expr.get(1).map_or(false, |x| match &x.kind {
+        ExprKind::Path(qp) => match_qpath(
+            qp,
+            &paths::OPTION
+                .iter()
+                .cloned()
```

Comment:
Isn't

```suggestion
                .copied()
```

enough here?